### DISCUSSION TITLE
Missing pkgs after download

### DISCUSSION THREAD

**Discussion by jamhouse2009** [bucanero/pkgi-ps3#36] [Q&A] [answered]
some games do not show up in package manager for install after completed download, HDD space goes down but the file is nowhere to be found.

games that have done this are: Dishonoured(US),Dead island Riptide(US), ghostbusters, wwe 12

**Comment by bucanero** [comment]
if you're using Background download, check the XMB Download manager.
In Direct download mode, check `/dev_hdd0/tmp/pkgi/`

**Comment by ghost** [comment]
Same issue happened for me. Tried both background and direct download. Neither worked. Using REBUG 4.82 DEX.

**Comment by bucanero** [comment, marked as answer]
tasks in background download mode will show up only after a reboot.

you can check these folders to see if there's any file there:
```
/dev_hdd0/tmp/pkgi/
/dev_hdd0/vsh/task/
/dev_hdd0/vsh/game_pkg/
```

**Comment by ghost** [comment]
Fixed it. Dumb mistake on my part, I didn't check the PSN install list.
